packages/react-native/Libraries/Text/TextInput/RCTBaseTextInputView.mm

```diff
@@ -914,8 +912,8 @@ - (void)setDefaultInputAccessoryView
     textInputView.inputAccessoryView = nil;
   }
   [self reloadInputViewsIfNecessary];
-#endif // [macOS]
 }
+#endif // [macOS]
 
 #if !TARGET_OS_OSX // [macOS]
```

Comment:
Should we just extend this ifdef even more? Accessory views are really only on iOS